hack/make-bundle-vars.mk

```diff
@@ -65,6 +65,11 @@ NOOBAA_BUNDLE_IMG_TAG ?= v5.9.0
 NOOBAA_BUNDLE_IMG_LOCATION ?= quay.io/noobaa
 NOOBAA_BUNDLE_IMG ?= $(NOOBAA_BUNDLE_IMG_LOCATION)/$(NOOBAA_BUNDLE_IMG_NAME):$(NOOBAA_BUNDLE_IMG_TAG)
 
+CSI_ADDONS_BUNDLE_IMG_TAG ?= v0.1.0
```

Comment:
The bundle image is  getting updated to `v0.1.1` after csi-addons/kubernetes-csi-addons#72 is merged. Manual testing (with slight modification to this PR) makes deploying csi-addons with odf-controller work for me.